orchestrator/src/main/kotlin/Orchestrator.kt

```diff
@@ -404,3 +408,13 @@ class Orchestrator(
  * Type definition to represent a list of jobs that have been created and must be scheduled.
  */
 typealias CreatedJobs = List<JobScheduleFunc>
+
+/**
+ * Create an [Issue] object representing an error that occurred in any [Endpoint].
+ */
+fun <T : Any> Endpoint<T>.createErrorIssue(): Issue = Issue(
+    timestamp = Clock.System.now(),
+    source = configPrefix,
+    message = "An unexpected failure occurred in the $configPrefix worker.",
```

Comment:
Maybe the message should be more drastic, e.g. something like "The $configPrefix worker failed due to an unexpected error." to make clear that this worker failed completely.